docs/brokerpak-intro.md

```diff
@@ -1,117 +1,65 @@
+* Service brokers serve the needs of the operators, developers, and platforms. They map the services out to match a variety of different businesses models, threat models, regulatory constraints, and use-cases.
 
-## Expression language reference
+Together, this means a service broker must:
 
-The broker uses the [HIL expression language](https://github.com/hashicorp/hil) with a limited set of built-in functions.
+* Provide lots of services for developers.
+* Provide services granular enough that operators can control cost.
+* Provide services robust enough that operators can control security.
+* Provide services structured enough operators can trust they'll be in compliance.
+* Provide services configurable enough developers will be happy.
+* Map a single platform service into its N use-cases so operators can grant developers fine-grained access.
+* Write documentation for each of those N use-cases.
+* Be backwards compatible with all changes so developers and operators get seamless upgrades.
+* Do all of the above at the rapid pace the platform grows at.
 
-## Function reference
+Some of these tasks can be automated, but many require deeper understanding of the platform and specific services to do correctly.
 
-The following string interpolation functions are available for use:
+### How brokerpak solves these problems
 
-* `assert(condition_bool, message_string) -> bool`
-  * If the condition is false, then an error will be raised to the user containing `message_string`.
-  * Avoid using this function. Instead, try to make it so your users can't get into a bad state to begin with. See the "design guidelines" section.  
-  * In the words of [PEP-20](https://www.python.org/dev/peps/pep-0020/):
-    * If the implementation is hard to explain, it's a bad idea.
-    * If the implementation is easy to explain, it may be a good idea.
-* `time.nano() -> string`
-  * This function returns the current UNIX time in nanoseconds as a decimal string.
-* `regexp.matches(regex_string, string) -> bool`
-  * Checks if the string matches the given regex.
-* `str.truncate(count, string) -> string`
-  * Trims the given string to be at most `count` characters long.
-  * If the string is already shorter, nothing is changed.
-* `counter.next() -> int`
-  * Provides a counter that increments once per call within the same call context.
-  * The counter is reset on restart of the application.
-* `rand.base64(count) -> string`
-  * Generates `count` bytes of cryptographically secure randomness and converts it to [URL Encoded Base64](https://tools.ietf.org/html/rfc4648).
-  * The randomness makes it suitable for using as passwords.
-* `json.marshal(type) -> string`
-  * Returns a JSON marshaled string of the given type.
+The brokerpak is a (zip) package that contains bundled versions of Terraform, service definitions (as Terraform modules), Terraform providers, and source code for regulatory compliance.
```

Comment:
Consider embedding a link to Terraform here, perhaps: https://www.terraform.io/intro/index.html